test/automated/ansible/group_vars/localhost/main.yml

```diff
@@ -25,20 +25,20 @@ instances:
     platform: "linux"
     python_interpreter: "/usr/bin/python3"
     launch_template: "LaunchTemplateId=lt-0b00afb3f5110a0e6,Version=3"
-  - ami: "ami-04511222dedb7385d"
-    type: "t3a.small"
-    name: "amd64:ubuntu21.04"
-    username: "ubuntu"
-    platform: "linux"
-    python_interpreter: "/usr/bin/python3"
-    launch_template: "LaunchTemplateId=lt-0b00afb3f5110a0e6,Version=3"
-  - ami: "ami-028f0daffc74d96ee"
-    type: "t3a.small"
-    name: "amd64:ubuntu20.10"
-    username: "ubuntu"
-    platform: "linux"
-    python_interpreter: "/usr/bin/python3"
-    launch_template: "LaunchTemplateId=lt-0b00afb3f5110a0e6,Version=3"
+#  - ami: "ami-04511222dedb7385d"
```

Comment:
I would add a comment to explain why this was commented (and if it's something that will not come back I would delete it :) )